<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnCnt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelByname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>名称查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDcnt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>断开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>插入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>261</width>
      <height>107</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="line_id"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>TNAME</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_int">
       <property name="text">
        <string>TINT</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="line_int"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="line_name"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_id">
       <property name="text">
        <string>TID</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_id_new">
       <property name="text">
        <string>OTID</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="line_id_old"/>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
